4Homework10.03.22/UniqueList/UniqueList.Tests/ListTests.cs

```diff
@@ -0,0 +1,67 @@
+using NUnit.Framework;
+
+namespace UniqueList.Tests;
```

Comment:
```suggestion
namespace UniqueList.Tests;

using NUnit.Framework;
```
Это не то чтобы прямо требование, но обычно пишут так